src/skmultiflow/meta/oza_bagging.py

```diff
@@ -159,12 +159,13 @@ def partial_fit(self, X, y, classes=None, weight=None):
                 raise ValueError("The classes passed to the partial_fit function differ from those passed earlier.")
 
         self.__adjust_ensemble_size()
-
-        for i in range(self.n_estimators):
-            k = self.random_state.poisson()
-            if k > 0:
-                for b in range(k):
-                    self.ensemble[i].partial_fit(X, y, classes, weight)
+        r, _ = get_dimensions(X)
+        for j in range(r):
```

Comment:
I don't think there is a bug. From the paper:

> For each base model h_m, (m in {1,2,....,M}) in the ensemble,
>   * Set *k* according to *Poisson(1)*
>   * Do *k* times 
>      *h_m = L_0(h_m, d)*

Could you elaborate on this?

Reply:
If I run this code:

```
stream = SEAGenerator()
stream.prepare_for_use()
learner= OzaBagging()
X, y = stream.next_sample(1000)
learner.partial_fit(X, y)
```
All the 1000 samples with have the same `k`, and the model will be trained `k` times on the 1000 samples. From what I know, `k` need to be recalculated for every sample.

Reply:
Oh, I see. You are right. Good catch.